notebooks/official/prediction/get_started_with_direct_raw_predict.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<p>Is this correct "Do to the requirements ..."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>